diff --git a/api-mispelis/db.js b/api-mispelis/db.js
index c467d4d..9180be7 100644
--- a/api-mispelis/db.js
+++ b/api-mispelis/db.js
@@ -24,7 +24,7 @@ export async function guardarPeli(peli) {
 
         const existente = await coleccion.findOne({
             usuario: peli.usuario,
-            titulo: peli.title || peli.titulo,
+            title: peli.title,
         });
 
         if (existente) {
@@ -68,7 +68,7 @@ export async function borrarPeli(id) {
         const db = conexion.db;
         const coleccion = db.collection("pelis");
 
-        const resultado = await coleccion.deleteOne({ _id: ObjectId(id) });
+        const resultado = await coleccion.deleteOne({ id: Number(id) });
 
         return resultado.deletedCount;
 
diff --git a/api-mispelis/index.js b/api-mispelis/index.js
index 018ff41..bf684f0 100644
--- a/api-mispelis/index.js
+++ b/api-mispelis/index.js
@@ -72,7 +72,7 @@ servidor.post("/pelivista", async (peticion, respuesta) => {
 
 
 // NO FUNCIONA! genera un id "undefined"
-servidor.delete("/borrarpeli/:id([0-9a-f]{24})", async (peticion, respuesta) => {
+servidor.delete("/borrarpeli/:id", async (peticion, respuesta) => {
     const { id } = peticion.params;
     console.log("ID recibido en el backend:", id);
     try {
diff --git a/front-mispelis/src/componentes/Acciones.jsx b/front-mispelis/src/componentes/Acciones.jsx
index 09326ce..af96c9a 100644
--- a/front-mispelis/src/componentes/Acciones.jsx
+++ b/front-mispelis/src/componentes/Acciones.jsx
@@ -17,7 +17,7 @@ export const Acciones = ({peli, tipo}) => {
                     <i className="fa-fw far fa-eye"></i>
                 </button>
 
-                <button className="ctrl-btn" onClick={() => borrarFav(peli._id)}>
+                <button className="ctrl-btn" onClick={() => borrarFav(peli)}>
                     <i className="fa-fw fa fa-times"></i>
                 </button>
                 </>
diff --git a/front-mispelis/src/contexto/Contexto.jsx b/front-mispelis/src/contexto/Contexto.jsx
index dbdba49..89de73a 100644
--- a/front-mispelis/src/contexto/Contexto.jsx
+++ b/front-mispelis/src/contexto/Contexto.jsx
@@ -45,16 +45,16 @@ export const GlobalProvider = (props) => {
     };
 
     // Elimina una peli de tipo "favoritas"
-    const borrarFav = (id) => {
-        console.log(id);
-        dispatch({ type: "BORRAR_FAV", payload: id });
+    const borrarFav = (peli) => {
+        console.log(peli);
+        dispatch({ type: "BORRAR_FAV", payload: peli.id });
 
-        fetch(`http://localhost:4000/borrarpeli/${id}`, {
+        fetch(`http://localhost:4000/borrarpeli/${peli.id}`, {
             method: "DELETE"
         })
         .then(respuesta => {
             if (respuesta.status === 204) {
-                dispatch({ type: "BORRAR_FAV", payload: id });
+                dispatch({ type: "BORRAR_FAV", payload: peli.id });
             } else {
                 return respuesta.json().then(error => console.log(error));
             }
